src/views/v2/commerce/checkout/purchaser-info/email.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Tickets Commerce: Checkout Page Purchaser Info Email Field
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/commerce/checkout/purchaser-info/email.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    https://evnt.is/1amp Help article for RSVP & Ticket template files.
+ *
+ * @since   TBD
+ *
+ * @version TBD
+ *
+ * @var \Tribe__Template $this [Global] Template object.
+ */
+
+$label_classes = [
+	'tribe-common-b3',
```

Comment:
It's lacking `tribe-tickets__form-field-label`. Are we sure we need a `tribe-common-b3`, we need to make this look like the designs?

Please refer to https://github.com/the-events-calendar/event-tickets-plus/blob/master/src/views/v2/components/meta/email.php to check the structure and generic classes.

Please [check the designs](https://www.figma.com/file/TvIXoUEJAGb0AptZI260UO/TicketsCommerce?node-id=1495%3A362) for better clarity.